doc/content/tutorials/restart_nek_moose.md

```diff
@@ -0,0 +1,55 @@
+# Restarting Coupled NekRS and MOOSE simulations
+
+This tutorial describes how to restart coupled NekRS
+and MOOSE simulations. First, the user needs to output
+restartable solution/checkpoint files for both NekRS and MOOSE.
+Then, the user needs to restart both simulations
+using said checkpoint files.
+
+# Creating NekRS checkpoint files
+
+NekRS checkpoint file output is controlled
+using the `writeControl` and `writeInterval` parameters in the .par file.
+The .par example below only writes a single checkpoint file
+at the end of the simulation.
+
+!listing /tutorials/restart_nek_and_moose/create_checkpoints/pyramid.par
+
+# Creating MOOSE checkpoint files
+
+MOOSE checkpoint file output is controlled
+using [Outputs/Checkpoint](https://mooseframework.inl.gov/source/outputs/Checkpoint.html).
+This is one of the [Restart and Recover](https://mooseframework.inl.gov/application_usage/restart_recover.html)
+options in the MOOSE framework. This checkpoint method requires N to N restarting, where the number of processors
+for the previous and current MOOSE simulations must match.
+
+!listing /tutorials/restart_nek_and_moose/create_checkpoints/cardinal_sub.i
+  block=Outputs
+
+# Reading NekRS checkpoint files
+
+The NekRS solution is restarted using the `startFrom`
+parameter in the NekRS .par file. `restartFrom` should contain the
+checkpoint file to restart the NekRS portion of the coupled simulation from.
+
+!listing /tutorials/restart_nek_and_moose/read_from_checkpoints/pyramid.par
+
+# Reading MOOSE checkpoint files
+
+The MOOSE solution is restarted within its Problem block
+using `restart_file_base`. This should contain the
+checkpoint file base to restart the MOOSE portion of the
+coupled simulation from. Furthermore, if MOOSE is running as the sub app in the MultiApp
+system, `force_restart` can be set to restart the MOOSE sub app
+without restarting the main app.
+
+!listing /tutorials/restart_nek_and_moose/read_from_checkpoints/cardinal_sub.i
+  block=Problem
+
+# Restart MOOSE at time zero
+
+By default, NekRS will restart its simulation from time zero. The `start_time` in MOOSE
```

Comment:
Just curious - does this cause a problem/inconvenience at all? (having NekRS start at zero time no matter what) We could probably change that, I just don't do that many restart calculations myself so haven't had to deal with the consequences of that choice.

Reply:
I think it can be confusing for users if they want to run a `NekRSStandaloneProblem` for example when using `startFrom`, time in Cardinal doesn't match the restart file's time.

Could we add a parameter to set the NekRS `start_time=0` if desired, maybe for all problem classes?

Then in the restart tutorial we'd set both NekRS `start_time=0` and MOOSE app `start_time=0`, to enforce the coupled simulation to restart together right away. I think it's good for control this for restarting coupled simulations.

Reply:
Would you please edit this PR to update the `start_time` discussion, now that we changed it in #388?

Thanks!